app/main/views/briefs.py

```diff
@@ -225,7 +225,7 @@ def redirect_to_next_page():
                     brief_response_id,
                     current_user.email_address
                 )
-
+                flash('submitted_first', 'success')
```

Comment:
`submitted_first` sounds a little unclear to me... saying that I don't think I have a better suggestion. The best I can think of is `submitted_on_previous_page`. Happy to keep as is if need be